### DISCUSSION TITLE
Support for multiple backup storage backends

### DISCUSSION THREAD

**Discussion by bethnull** [Archmonger/django-dbbackup#580] [Feature Requests]
It would be nice to have the ability of using multiple backup storage backends so the backups will be stored in multiples places.

For instance, backing up to `django.core.files.storage.FileSystemStorage` and `storages.backends.s3boto.S3BotoStorage` at the same time.

By the way, django-dbbackup rocks! ;)

BR, Pepe.

**Comment by ZuluPro** [comment]
Hello @bethnull 
Thanks for the suggestion. We are already think about something like that in #61:
But we didn't think about a simple multi-storage mode.

I will we included soon I hope, but just one question:
How do you think it will deal with files ? Its reference will be only the first storage ?

**Comment by erasmodanarni** [comment]
It would be a great addition to the functionalities of the package!

**Comment by racitup** [comment]
This would be great.
I was thinking you could just add a layer to the backend configuration a lot like the django database backend configuration. 'default' would be the default but people could add others that are selectable from a command option. For example 'local', 's3' and 'dropbox'.
At the moment I have solved this by loading different settings files. Useful for restoring a media backup for initial deployment locally and offside db backups in normal operation.

**Comment by ZuluPro** [comment]
@racitup 
Yes I was thinking exactly about the same thing ;)
I just don't have the time, maybe you could propose a PR ?

**Comment by racitup** [comment]
I wish I could but I don't have the time either 😦

**Comment by mjlabe** [comment]
If I find some time (probably not soon), I would like to take a look. 

One feature I would like is a flag to pick the storage, so daily backups could be local, but weekly backups would be local and S3. Or implementing as @ZuluPro suggested in #61.
